### ISSUE TITLE
serialized.indexOf is not a function

### ISSUE THREAD

**Issue by Sgajjar08** [coopermaruyama/tableau-react#28]
I am always having this error when I use tableau-react in my react project.

**Comment by tcjcodes** [comment]
Full error:

```
tableau-2.0.0.js?b650:731 Uncaught TypeError: serialized.indexOf is not a function
    at Function.tab__ApiCommand$parse [as parse] (tableau-2.0.0.js?b650:731)
    at tab__crossDomainMessageRouter$_handleCrossDomainMessage [as _handleCrossDomainMessage] (tableau-2.0.0.js?b650:1025)
    at tab__crossDomainMessageRouter.eval (tableau-2.0.0.js?b650:1017)
    at delegate (tableau-2.0.0.js?b650:181)
```

Versions:
```json
    "tableau-api": "^2.2.3",
    "tableau-react": "^1.2.2"
```

**Comment by PurpleMonkeyWrench** [comment]
How can this change be moved out of beta and into the latest version?  Also, it looks like there is a pull request to update the tableau api to 2.7.0.  Would pulling in that request also fix this issue?

**Comment by mcdowellalex** [comment]
This is still an issue

**Comment by coopermaruyama** [comment]
I've published 2.0.0 -- please test it and let me know if it works
